modules/classes/sequencing.yaml

```diff
@@ -1,15 +1,23 @@
 classes:
   
-  Sequencing:
+  File:
```

Comment:
Wondering if we want to change imaging to inherit from "File" as well.

Reply:
Might not be as easy as I think though..

Reply:
OK after trial and errors..I just changed LabID to ParticipantID to be consistent with the other templates..I think that's all for now - 

General question on the inheriting - Can we do "is_a" across different yaml files? (Like have imaging inherit from File if it's in another yaml file?)

Reply:
OK, thanks for updating LabID to make it more consistent. Yep, can do `is_a` across different files.

Reply:
We'll apply the refactoring for imaging in another PR to keep this one smaller.